rust-webvr/src/api/googlevr/gradle/src/main/java/com/rust/webvr/GVRService.java

```diff
@@ -102,6 +108,10 @@ public void run() {
                 // Hide GvrLayout
                 FrameLayout rootLayout = (FrameLayout)mActivity.findViewById(android.R.id.content);
                 rootLayout.removeView(gvrLayout);
+
+                AndroidCompat.setVrModeEnabled(mActivity, false);
+                mActivity.setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_USER_PORTRAIT);
```

Comment:
What if the user was browsing in landscape before he enters VR? Should we save the last orientation before entering VR?